libs/libarchfpga/src/physical_types.h

```diff
@@ -1382,6 +1382,14 @@ enum e_directionality {
     UNI_DIRECTIONAL,
     BI_DIRECTIONAL
 };
+/* X_AXIS: Assigned to a segment's  parallel_axis data member if segment is parallel to the x-axis. Defined in arch file as: axis="x"           *
+ * Y_AXIS: Assigned to a segment's  parallel_axis data amember if segment is parallel to the y-axis. Defined in arch file as: axis="y"          *     
+ * BOTH_AXIS: Assigned to a segment's parallel axis data member if segment's parallel axis is not specified (default is it parallel to both).   */
+enum e_parallel_axis {
+    X_AXIS,
```

Comment:
I think you can still explain this a bit more, in a less self-referential way.  
X_AXIS: data that describes an x-directed wire segment (CHANX)
Y_AXIS: data that describes a y-directed wire segment (CHANY)
BOTH_AXIS: data that can be applied to both x- and y-directed wire segments.